app/mailers/conversation_reply_mailer_helper.rb

```diff
@@ -19,6 +19,33 @@ def prepare_mail(cc_bcc_enabled)
 
     Rails.logger.info("Email sent from #{email_from} to #{to_emails} with subject #{mail_subject}")
 
+    if @message.attachments.present?
```

Comment:
To fix [this](https://github.com/chatwoot/chatwoot/pull/9655#issuecomment-2384170806) just edit this line `if @message&.attachments.present?`